packages/charts/react-charting/src/components/DonutChart/DonutChart.base.tsx

```diff
@@ -354,7 +374,7 @@ export class DonutChartBase extends React.Component<IDonutChartProps, IDonutChar
    * Note: This won't work in case of multiple legends selection.
    */
   private _getHighlightedLegend() {
-    return this.state.selectedLegend || this.state.activeLegend;
```

Comment:
> return this.state.selectedLegend || this.state.activeLegend; [](http://example.com/codeflow?start=3&length=61)

This is an or condition between selected and active.

Reply:
For multiple select also checking if this.state.selectedLegends is populated other return the this.state.activeLegend
    return this.state.selectedLegends.length > 0 ? this.state.selectedLegends : [this.state.activeLegend];